Sources/Components/SuggestShippingView/SuggestShippingView.swift

```diff
@@ -0,0 +1,139 @@
+import Combine
+import FinniversKit
+
+public final class SuggestShippingView: UIView {
+    private var viewModel: SuggestShippingViewModel!
+    private var cancellables = Set<AnyCancellable>()
+    private var buttonCancellable: AnyCancellable?
+
+    private let imageHorizontalInset: CGFloat = .spacingS
+
+    private let horizontalContainer: UIStackView = {
+        let stackView = UIStackView(axis: .horizontal)
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private let containerView: UIStackView = {
+        let stackView = UIStackView(axis: .vertical)
+        stackView.spacing = .spacingM
+        return stackView
+    }()
+
+    private let textContainer: UIStackView = UIStackView(axis: .vertical)
+
+    private lazy var title: Label = {
+        let label = Label(style: .captionStrong)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var message: Label = {
+        let label = Label(style: .caption)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var suggestShippingbutton: Button = {
+        let button = Button(style: .suggestShippingStyle)
```

Comment:
Use this instead, so we can remove the code on line 53
```suggestion
        let button = Button(style: .suggestShippingStyle, withAutoLayout: true)
```